src/make/createQualificationReport.m

```diff
@@ -0,0 +1,51 @@
+function createQualificationReport(qualificationWorkflow)
```

Comment:
new function (required for compiled mode only): sets the environment and executes `qualificationWorkflow `linewise via `eval`.